api/app/Console/Kernel.php

```diff
@@ -15,7 +19,40 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        $schedule->call(function () {
+            $deleteDate = Carbon::now()->subYears(5);
+            $users = User::onlyTrashed()->whereDate('deleted_at', '<=', $deleteDate)->get();
+
+            foreach ($users as $user) {
+                // Cascade delete to child models
```

Comment:
This is an interesting approach.  I think in other places we've tended to use database delete cascades instead of manually removing all the children first.  Is there a reason you went with this approach?

Reply:
No I didn't have any good reason for this, I've updated to use automatic database cascades with [ed961ae](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7593/commits/ed961ae4e258c556bf5a2b2bf7842fc5e127caee).